content/sensu-go/5.15/api/overview.md

```diff
@@ -135,6 +135,93 @@ http://127.0.0.1:8080/api/core/v2/namespaces/default/events
 If your token expires, you should see a 401 Unauthorized response from the API.
 To regenerate a valid access token, first run any sensuctl command (like `sensuctl event list`) then repeat step 2.
 
+### Authenticate with the API key feature
+
+The Sensu API key feature (core/v2.APIKey) is a persistent UUID that maps to a stored Sensu username. API keys are cluster-wide resources, so only cluster admins can grant, view, and revoke them.
+
+_**NOTE**: API keys are not supported for authentication providers such as LDAP and OIDC._
+
+### API key authentication
+
+Similar to the `Bearer [token]` Authorization header, `Key [api-key]` will be accepted as an Authorization header for authentication.
+
+For example, a JWT `Bearer [token]` Authorization header might be:
+
+{{< highlight shell >}}
+curl -H "Authorization: Bearer $SENSU_TOKEN" http://127.0.0.1:8080/api/core/v2/namespaces/default/checks
+{{< /highlight >}}
+
+If you're using `Key [api-key]` to authenticate instead, the Authorization header might be:
+
+{{< highlight shell >}}
+curl -H "Authorization: Key $SENSU_API_KEY" http://127.0.0.1:8080/api/core/v2/namespaces/default/checks
+{{< /highlight >}}
+
+#### Example
+
+{{< highlight shell >}}
+$ curl -H "Authorization: Key 7f63b5bc-41f4-4b3e-b59b-5431afd7e6a2" 
```

Comment:
This curl command appears incomplete, probably needs a URL added, e.g. `http://127.0.0.1:8080/api/core/v2/namespaces/default/checks`